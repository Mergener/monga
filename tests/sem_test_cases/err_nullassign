function main() {
    var i: int;
    i = null;
}