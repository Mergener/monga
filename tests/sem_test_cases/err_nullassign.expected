[Error] Module tests/sem_test_cases/err_nullassign, line 3, column 14: Cannot assign value of type '<null-type>' to lvalue of type 'int'.
Semantic errors were caught during analysis.
